tests/execution/test-cors-tailcall-run.md

```diff
@@ -0,0 +1,32 @@
+# Verify the correct CORS headers for requests from https://tailcall.run
+
+```graphql @config
+schema
+  @upstream(baseURL: "http://example.com")
+  @server(
+    port: 8000
+    headers: {
+      cors: {allowHeaders: ["*"], allowMethods: ["GET", "POST", "OPTIONS"], allowOrigins: ["https://tailcall.run"]}
+    }
+  ) {
+  query: Query
+}
+
+type Query {
+  example: String @http(path: "/example")
+}
+```
+
+```yml @test
+# CORS test
+- method: OPTIONS
```

Comment:
If cors is enabled we need to try to make requests with `GET` and `POST` methods also.